docs/userguide.md

```diff
@@ -123,6 +123,7 @@ Each score is documented in the API documentation [ api.md ](api.md). A simple l
 
  - Mean Absolute Error
  - Mean Squared Error
+ - Root Mean Squared Error
```

Comment:
For some reason I can't enter this comment on the right line - please also remove the entry from "the following scores are expected to be added shortly" a few lines down.